### PR TITLE
docs(#27) - Adding readme to One Platform

### PR DESCRIPTION
**Pull request by hybridx**
## PR Checklist
Please check if your PR fulfils the following requirements:

## PR Type
What kind of change does this PR introduce?
- [x] Documentation content changes

## What is the current behaviour?
Does not have a proper README

Issue Number: #27 

## Does this PR introduce a breaking change?
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,33 @@
+![Version](https://img.shields.io/badge/version-0.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+[![License: MIT](https://img.shields.io/github/license/1-Platform/one-platform)](https://github.com/1-Platform/one-platform/blob/master/LICENSE)
+
 <p align="center"><img src="https://avatars1.githubusercontent.com/u/58499608?s=100&v=4" alt="One Platform Logo"></p>
 
 # One Platform
```
Comment (deshmukhmayur) [inline README.md]: Move these lines to the top of the file.

The shields/badges should be below the heading.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
 An integrated application hosting platform.
```
Comment (deshmukhmayur) [inline README.md]: Can you add more detailed description about One Platform?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,36 @@
...
-An integrated application hosting platform.
+An integrated application hosting platform which allows you to host your SPAs. It also provides common services like
+- Authentication and Authorization
+- Notifications Framework
+- Feedback Framework
+
+![Version](https://img.shields.io/badge/version-0.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+[![License: MIT](https://img.shields.io/github/license/1-Platform/one-platform)](https://github.com/1-Platform/one-platform/blob/master/LICENSE)
```
Comment (deshmukhmayur) [inline README.md]: Can you move these after line 3.

The shields should be below the project title (i.e. the heading)

Comment (hybridx) [reply README.md]: Screenshot from 2020-02-28 12-02-15
Done

File: README.md [hunk 4]
```diff
@@ -1,4 +1,42 @@
...
+
+
+### 🏠 [Homepage](https://github.com/1-Platform/one-platform#readme)
+
+## Install
+
+```bash
+npm install
+```
+
```
Comment (deshmukhmayur) [inline README.md]: Can you add lerna bootstrap command here (https://lerna.js.org/#command-bootstrap)

Comment (hybridx) [reply README.md]: got it

**Pull request merged**